web/src/featureToggle.ts

```diff
@@ -1,19 +1,29 @@
 /* eslint no-console: 0 */
-/* eslint no-throw-literal: 0 */
 import Vue from 'vue';
+import { reactive } from '@vue/composition-api';
+
+// Declare additions to the global `window` object
+declare global {
+  interface Window {
+    enable: (toggle: string) => void,
+    disable: (toggle: string) => void,
+  }
+}
 
 // Create a component to hold the feature toggles reactively
 // This object can be mutated at runtime to enable/disable feature toggles
-const featureToggles = new Vue({
-  data() {
-    return {
-      DJANGO_API: false,
-    };
-  },
+const featureToggles = reactive({
+  DJANGO_API: false,
 });
 
+// Used to narrow string types
+function isToggle(toggle: string): toggle is keyof (typeof featureToggles) {
+  return Object.keys(featureToggles).includes(toggle);
```

Comment:
I think you can use `.hasOwnProperty()` here to avoid a listifcation and a linear-time search.

Reply:
You're correct. I originally didn't use that due to eslint warnings, but they seem not to apply here, so I'll fix it.